src/styles/Profile.css

```
@@ -16,4 +16,10 @@
     font-size: 40px;
     margin: 15px;
     color: black;
+}
+.replit-icon {
+    font-size: 40px;
+    margin: 15px;
+    margin-bottom: 67px;
```

As you can see, `margin-bottom` is a hardcoded value. I had to do some trial and error to get the right amount of margin. 

It shouldn't affect responsiveness, I tested it out on all the available viewports and it seems to be alright.

Also, the reason why I had to add an entirely new style for the Replit icon is because the icon providers (FontAwesome and React Icons) use different components and therefore, the icons aren't aligned as expected and therefore, I had to come up with this hacky workaround.